sormas-backend/src/main/java/de/symeda/sormas/backend/info/InfoFacadeEjb.java

```diff
@@ -241,6 +243,10 @@ private void createEntitySheet(XSSFWorkbook workbook, Class<? extends EntityDto>
 				continue;
 			XSSFRow row = sheet.createRow(rowNumber++);
 
+			// filedId
+			XSSFCell fieldIdCell = row.createCell(EntityColumn.FIELD_ID.ordinal());
+			fieldIdCell.setCellValue(i18nPrefix + "." + field.getName());
```

Comment:
I would say `DataHelper.getHumanClassName(entityClass)` instead of `i18nPrefix + "." + field.getName()` because `getHumanClassName` is used when generating the import template files and it's also used in other import related code.